app-frontend/src/assets/styles/sass/_shame.scss

```diff
@@ -3357,6 +3358,10 @@ rf-project-publishing-page {
     &.cozy {
         margin-bottom: 1.5rem;
     }
+    &.filter-pane {
```

Comment:
You can drop this class as explained in another comment.

Reply:
Resolved in https://github.com/raster-foundry/raster-foundry/pull/4733/commits/32aea7b0aa17f69614ec7a1feb2cb5480d58f712